packages/sanity/src/core/studio/workspaceLoader/WorkspaceLoader.tsx

```diff
@@ -158,9 +158,6 @@ function WorkspaceLoaderBoundary({ConfigErrorsComponent, ...props}: WorkspaceLoa
   // TODO: implement this
   // const errors = useMemo(() => flattenErrors(error, []), [error])
 
-  //TODO: implement config error screen
```

Comment:
I think we should keep this comment as it is a way to prepare for the actual implementation. This is the right place for the component to be rendered, so the comment will be helpful once we implement this screen for real in the future!

Reply:
fair enough, I'll add the same comment to this as we have on the configerrorscreen 👍